docker-gc

```diff
@@ -241,11 +242,23 @@ xargs -n 1 $DOCKER inspect -f '{{.Image}}' 2>/dev/null |
 sort | uniq > images.used
 
 # List images to reap; images that existed last run and are not in use.
-$DOCKER images -q --no-trunc | sort | uniq > images.all
+echo -n "" > images.all
+$DOCKER images | while read line
+do
+    awk '{print $1};'
+done | sort | uniq | while read line
+do
+  $DOCKER images --no-trunc --format "{{.ID}} {{.CreatedAt}}" $line \
+    | sort --key 2 --reverse \
```

Comment:
if the output from `docker images --no-trunc --format "{{.ID}} {{.CreatedAt}}"` looks something like:

```
sha256:abc123 2016-10-05 14:38:43 +0000 UTC
sha256:def798 2016-12-12 20:41:34 +0000 UTC
```

won't `sort --key 2` end up sorting by just the `yyyy-MM-dd` date, discarding the timestamp portion of `{{.CreatedAt}}`?